ericus123/PORTFOLIO_PSQL
# Project setup and deployment to heroku

#### What does this PR do?
Initial setup of the project including several initial configuration 
like babel, sequelize ORM and deploy the base app to heroku
#### Description of Task to be completed?
- Configure sequelize
- Configure babel
- Add default routes
- Deploy the app to heroku
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/ericus123/PORTFOLIO_APP `
- Switch directory `cd PORTFOLIO_APP`
- Checkout branch `git checkout ch-project-setup`
- Run migrations `npm run migrate`
> This will run migrations and can be seen in the database
- Run dev server `npm run dev`
>This will start the development server and connect to the postgres db
- Visit the hosted version [here](https://amanieric-portfolio-staging.herokuapp.com/)
>You should see a welcome message 
 
#### Any background context you want to provide?
N/A
#### Screenshots (if appropriate)
![welcome msg](https://user-images.githubusercontent.com/58434968/108338704-3c679880-71df-11eb-83d3-cacdf13a3c92.PNG)

#### Questions:
N/A
